argh/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "argh"
+version = "0.1.5"
+authors = ["Taylor Cramer <cramertj@google.com>", "Benjamin Brittain <bwb@google.com>", "Erick Tryzelaar <etryzelaar@google.com>"]
+edition = "2018"
+keywords = ["args", "arguments", "derive", "cli"]
+license = "BSD-3-Clause"
+description = "Derive-based argument parser optimized for code size"
+repository = "https://github.com/google/argh"
+readme = "README.md"
+include = ["Cargo.toml", "src/**/*.rs", "README.md", "LICENSE"]
```

Comment:
Is it not possible to put `../` in front of README.md in the Cargo.toml file instead of creating those dummy wrapper files in each subdirectory?

Reply:
Unfortunately cargo filters out parent files from `include` (which is sensible to avoid stealing outside files), but doesn't warn on this. I filed https://github.com/rust-lang/cargo/issues/9667 to fix that.

But as looking into this, I see cargo now has a `license-file` that points at the license, which is a bit more discoverable than just including it with `include`. I'll switch to that.